chunky/src/java/se/llbit/chunky/renderer/RenderManager.java

```diff
@@ -39,12 +39,12 @@ public interface RenderManager {
   /**
    * Get all available {@code Renderer}s.
    */
-  Collection<? extends Registerable> getRenderers();
+  Collection<Renderer> getRenderers();
```

Comment:
This would make it impossible for plugins to implement `RenderManager` and have their own type of swappable renderer that is incompatible with Chunky's renderers. (Which kinda defeats the point of changing the RenderManager implementation).

Reply:
Oh… Okay, so a doesn't have to be a `Renderer`?

Reply:
Yeah, `Renderer` is just the interface to work with Chunky's `DefaultRenderManager`.